src/common/components/header/header.scss

```diff
@@ -0,0 +1,49 @@
+.navbar {
+  width: 100%;
+  height: 80px;
+  left: 200px;
+  top: 220px;
+  background: #fff;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .navbar--wrapper {
```

Comment:
Since `wrapper`, `logo` and `items` are elements within the `navbar` block, by following the BEM convention, the class namings would change into `.navbar__wrapper`, `.navbar__logo` etc. 
Also, SASS allows to simplify the syntax from `.navbar__wrapper` into just `&__wrapper`. The '&' operator makes a concatenation and the result is the same, but cleaner and makes it easier to understand the structure.

Reply:
On point! I'll use the &__element syntax from now on. It really is easier to understand and to write. Thank you!